handler/core/src/main/java/io/knotx/fragments/handler/FragmentsHandlerOptions.java

```diff
@@ -31,6 +31,10 @@
 
   private List<FactoryOptions> consumerFactories = Collections.emptyList();
 
+  private String allowInvalidFragmentsHeader;
```

Comment:
Please update the documentation: https://github.com/Knotx/knotx-fragments/tree/master/handler#fragment-execution-log-consumer. Handler loads all configured consumers, then if this option is configured, then consumer gets data even if some fragments failed. This should be in this section.